spec/lib/openstax/auth/strategy_2_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+require 'openstax/auth/strategy_2'
+
+RSpec.describe OpenStax::Auth::Strategy2 do
+
+  SIGNATURE_PUBLIC_KEY = <<~SIGNATURE
+      -----BEGIN PUBLIC KEY-----
+      MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAxItufComL1S+j+U2JVr4
+      aeIPyZYJR8S3RcxXjlHgybrAsBf/zyAjJlOBIhVfrm9jUF1HKNMyiTKQehG5XBBW
+      /F8DQT5vGdBd4g9WIAmNU0E+symeF4X+mFIZ6dYwTMKtZxv1U0nkJ8xL2q4jCFVB
+      UMlQRmz9EffSz+fmXr9xGQj80HKahzciM6m2aspX096qUP90155qmLEayE2uhs5C
+      oAUbahA+VXS6ggsCUeVyog5Z1mC086d8r78ylr1y8IQ3aazdJE/mKxfqvu9S423h
+      wNzBP6Fp0n68ZjUdUirqAZEbSrioJgFLEzX8ef7XilTL9dKLSS1w09ErctAF+Tor
+      hwIDAQAB
+      -----END PUBLIC KEY-----
+  SIGNATURE
+
+  SECRETS = {
+    # these values copied from the Accounts secrets
```

Comment:
to not give future devs a heart attack, can you clarify here that these are copied from the dummy development secrets? :-)